.github/scripts/identify-builds-from-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# If the string `[builds-from-run: <run-id>]` is contained in the last commit message
```

Comment:
thinking: instead of making the user manually commit a specific message, could we read from the `changed-files/changed-files.json` and perform that logic automatically (if no app files changed, then use existing build from main) ?